### PR TITLE
Sort python requirements files

### PR DESCRIPTION
**Pull request by portante**
Broken out from PR #2551.

### CODE REVIEW

**Comment by portante** [comment]
Rebased on PR #2646 to provide a layering for review of the final work in #2551.

File: lib/pbench/common/selinux.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    import selinux
+except ImportError:
+
+    class selinux:
+        """Pbench SELinux wrapper."""
+
+        @staticmethod
+        def is_selinux_enabled():
+            """Always returns False."""
+            return False
```
Comment (dbutenhof) [inline lib/pbench/common/selinux.py:30]: Is this what you want? My investigations when I wrote the file tree mocks showed that the real selinux returns integer `0` or `1`, not `True` or `False`.

**Review by dbutenhof — CHANGES_REQUESTED** [review]

**Review by webbnh — APPROVED** [review]
I'm not sure what it means to approve a draft PR, but I'm gonna try it....

**Review by dbutenhof — DISMISSED** [review]

**Comment by portante** [comment]
> I'm not sure what it means to approve a draft PR, but I'm gonna try it....

The goal is to make sure this PR does not get submitted with the commit that is taken from PR #2646.  Once that is merged, then we can consider merging this one.

Boy do I wish we had Gerrit.

**Comment by webbnh** [comment]
> The goal is to make sure this PR does not get submitted with the commit that is taken from PR #2646.

That's fair.

I wanted to see whether my approval got revoked when you marked this ready to review, but your rebase cleared it....

**Review by portante — COMMENTED** [review]
Okay, ready for review.

**Review by dbutenhof — APPROVED** [review]

**Review by webbnh — APPROVED** [review]
👍

**Pull request merged**